test/unit/style.spec.js

```diff
@@ -0,0 +1,23 @@
+/* global $,testWrite,setOptions */
+/* eslint-disable no-var */
+import postscribe from '../../dist/postscribe';
```

Comment:
I don't think you want to depend on the dist, do you?  Won't `webpack` pick up the source and do the right thing?